generators/app/templates/src/app/screens/OnBoarding/components/Swiper/screens.js

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+
+import FirstScreen from './components/FirstScreen';
+import SecondScreen from './components/SecondScreen';
+import ThirdScreen from './components/ThirdScreen';
+
+const screenNames = {
+  FirstScreen: 'FirstScreen',
+  SecondScreen: 'SecondScreen',
+  ThirdScreen: 'ThirdScreen'
+};
+
+export const screensComponents = {
+  [screenNames.FirstScreen]: <FirstScreen />,
+  [screenNames.SecondScreen]: <SecondScreen />,
+  [screenNames.ThirdScreen]: <ThirdScreen />
+};
```

Comment:
is this necessary? why not just defining an array with the components? and use the `index` value of the swiper to render the correct screen

using the index will make easier to add/remove/change a screen.

Reply:
I like the idea